core/tests/webdriverio_utils/ExplorationEditorSettingsTab.js

```diff
@@ -139,26 +159,27 @@ var ExplorationEditorSettingsTab = function() {
 
   this.expectCategoryToBe = async function(category) {
     await waitFor.presenceOf(
-      explorationCategoryInput,
+      explorationCategory,
       'Exploration category input takes too long to be visible.');
-    expect(await explorationCategoryInput.$('option:checked').getText()).
+    expect(await $('.e2e-test-exploration-category-dropdown').getText()).
```

Comment:
Instead of directly using the `$` selector, please create a variable at the top and then use `getText()` on it. 
Ditto elsewhere.

Reply:
done